tests/console/commands/source/test_add.py

```diff
@@ -32,7 +32,12 @@ def assert_source_added(
 ) -> None:
     assert tester.io.fetch_error().strip() == ""
     assert (
-        tester.io.fetch_output().strip()
+        tester.io.fetch_output()
+        .strip()
+        .replace(
+            "The default priority will change to supplemental in a future release.\n",
+            "",
+        )
```

Comment:
I would prefer testing that the warning is included.

Reply:
😅 I just took the quick way out. Will add a separate test case for it.